indra/newview/llappcorehttp.cpp

```diff
@@ -281,6 +283,15 @@ void LLAppCoreHttp::init()
         LL_INFOS("Init") << "HTTP Pipelining " << (mPipelined ? "enabled" : "disabled") << "!" << LL_ENDL;
     }
 
+    // Global multiplexing setting
+    static const std::string http_multiplexing("UseHTTP2Multiplexing");
```

Comment:
Personally I would have merged "HttpPipelining" and "UseHTTP2Multiplexing" into some kind of 'mode switch' since they are sort of exclusive.